sycl/doc/SYCLEnvironmentVariables.md

```diff
@@ -4,11 +4,19 @@ This file describes environment variables that are having effect on SYCL compile
 
 # Controlling SYCL RT
 
-| Environment variable | Description |
-| ----------- | ----------- |
-| SYCL_PI_TRACE | If set forces tracing of PI calls to stdout. |
-| SYCL_BE={PI_OPENCL,PI_OTHER} | When SYCL RT is built with PI this controls which plugin to use. |
-| SYCL_PRINT_EXECUTION_GRAPH | Print execution graph to DOT text file. Options are described below. |
+| Environment variable | Values | Description |
+| -------------------- | ------ | ----------- |
+| SYCL_PI_TRACE | Any(*) | Force tracing of PI calls to stderr. |
+| SYCL_BE | PI_OPENCL, PI_OTHER | When SYCL RT is built with PI this controls which plugin to use. Default value is PI_OPENCL. |
+| SYCL_DEVICE_TYPE | One of: CPU, GPU, ACC, HOST | Force SYCL to use the specified device type. If unset, default selection rules are applied. If set to any unlisted value, this control has no effect. If the requested device type is not found, a `cl::sycl::runtime_error` exception is thrown. If a non-default device selector is used, a device must satisfy both the selector and this control to be chosen. |
```

Comment:
The wording is not accurate, SYCL_DEVICE_TYPE makes so only devices with specified type are passed to sycl::device_selector::operator(). There are some places where device_selector is not used.

Also I have a concern about documenting this variable as users can start rely on it. 
This variable was introduced for developers only(and to simplify testing), it has disadvantages like: it violates the spec because host device should always be available, it's still possible to create device of different type, it's not aligned with other SYCL implementations, it allows make more find grained selection(like specify) platform name, it has no diagnostics on incorrect usage.